components/scream/scripts/test_all_scream.py

```diff
@@ -552,10 +558,17 @@ def run_test(self, test):
         cmake_config = self.generate_cmake_config(self._tests_cmake_args[test], for_ctest=True)
         ctest_config = self.generate_ctest_config(cmake_config, [], test)
 
-        # This directory might have been used also to build the model to generate baselines.
-        # Although it's ok to build in the same dir, we MUST make sure to erase cmake's cache
-        # and internal files from the previous build (CMakeCache.txt and CMakeFiles folder)
-        run_cmd_no_fail("rm -rf CMake*", from_dir=test_dir)
+        if self._quick_rerun:
+            # Do not purge bld dir, and do not rerun config step.
+            # Note: make will still rerun cmake if some cmake file has changed
+            ctest_config += "-DSKIP_CONFIG_STEP=TRUE "
+            if self._quick_rerun_failed:
+              ctest_config += "--rerun-failed "
```

Comment:
Mixing 2 and 4-space indents. We usually use 4 for python.

Reply:
Ah, good catch. I tried to set all indent to 4 (even though my vim uses 2), but I missed this. Thanks.

Ps: I didn't know about pylint. I see that sems python modules don't make it available. Is it a separate module/package?